src/slim-select/index.ts

```diff
@@ -482,4 +486,23 @@ export default class SlimSelect {
       document.body.removeChild(slimContent)
     }
   }
+
+
+  private documentClick(e: Event): void {
+    if (e.target && !hasClassInTree(e.target as HTMLElement, this.config.id)) {
+      this.close()
+    }
+  }
+
+  private windowScroll(e: Event): void {
+      console.log(this.slim.container)
+      if (this.data.contentOpen) {
+          if (putContent(this.slim.content, this.data.contentPosition, this.data.contentOpen) === 'above') {
+              this.moveContentAbove()
+          } else {
+              this.moveContentBelow()
+          }
+      }
+  }
```

Comment:
to match the previous comment. note that you also had left a `console.log` there.

```suggestion
  private windowScroll: (e: Event) => void = debound(e => {
      if (this.data.contentOpen) {
          if (putContent(this.slim.content, this.data.contentPosition, this.data.contentOpen) === 'above') {
              this.moveContentAbove()
          } else {
              this.moveContentBelow()
          }
      }
  })
```